src/main/java/com/lgtm/easymoney/controllers/AnalyticController.java

```diff
@@ -0,0 +1,38 @@
+package com.lgtm.easymoney.controllers;
+
+import com.lgtm.easymoney.payload.rsp.AnalyticRsp;
+import com.lgtm.easymoney.security.CurrentUser;
+import com.lgtm.easymoney.security.UserPrincipal;
+import com.lgtm.easymoney.services.AnalyticService;
+import com.lgtm.easymoney.services.SearchService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.Parameter;
+import javax.validation.constraints.NotNull;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Analytic controller.
+ */
+@RestController
+@RequestMapping("/analytic")
+public class AnalyticController {
+  private final AnalyticService analyticService;
+
+  @Autowired
+  public AnalyticController(AnalyticService analyticService) {
+    this.analyticService = analyticService;
+  }
+
+  @GetMapping
```

Comment:
Please add the swagger security annotation here (see how other controllers are written)